src/_capi.pyx

```diff
@@ -40,7 +40,7 @@ cdef uint32_t _get_rgba32_value(color):
     else:
         alpha = 255
 
-    return (alpha << 24) | (r << 16) | (g << 8) | b
+    return (alpha << 24) | (b << 16) | (g << 8) | r
```

Comment:
Now I'm curious if my implementation always had this wrong or if it changed upstream? 😅

Reply:
I think it was wrong because initially without updating blend2d my blue lines where orange. I must say I was a bit suprised !